### PR TITLE
feat: add databend rust testcontainer docs

### PR DESCRIPTION
**Pull request by hantmac**
Ref: https://github.com/testcontainers/testcontainers-rs-modules-community/pull/207

### CODE REVIEW

**Comment by hantmac** [comment]
@mdelapenya Would you like to review it ?

File: modules/databend/index.md [hunk 1]
```diff
@@ -31,6 +31,16 @@ docs:
       ```bash
       go get github.com/testcontainers/testcontainers-go/modules/databend
       ```
+  - id: rust
+    url: https://github.com/testcontainers/testcontainers-rs-modules-community
```
Comment (mdelapenya) [inline modules/databend/index.md:35]:
```suggestion
    url: https://github.com/testcontainers/testcontainers-rs-modules-community/tree/main/src/databend
```

File: modules/databend/index.md [hunk 2]
```diff
@@ -31,6 +31,16 @@ docs:
...
+  - id: rust
+    url: https://github.com/testcontainers/testcontainers-rs-modules-community/tree/main/src/databend
+    maintainer: community
+    example: |
+    ```rust
+     let databend = DatabendImage::default().start().await.unwrap();
+    ```
+    installation: |
+    ```rust
+    use testcontainers_modules::{databend::Databend as DatabendImage, testcontainers::runners::AsyncRunner};
```
Comment (DDtKey) [inline modules/databend/index.md:43]: I believe this should be part of `example`.

While installation is:

```
cargo add -F databend --dev testcontainers-modules
```

Comment (mdelapenya) [reply modules/databend/index.md:43]: @hantmac could you check the changes in #79 ? 🙏

Comment (hantmac) [reply modules/databend/index.md]: @mdelapenya Sorry for late response, I fix it according to surrealdb module docs.

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

**Pull request merged**